specs/commandline/blob.md

```diff
@@ -342,6 +357,60 @@ The trust policy configuration in the JSON file should be validated according to
 
 If there is an existing trust policy configuration, prompt for users to confirm whether discarding existing configuration or not. Users can use `--force` flag to discard existing trust policy configuration without prompt.
 
+### Initialize blob trust policy configuration
+
+The following command initializes a blob trust policy configuration:
+
+```shell
+notation blob policy init --name "myBlobPolicy" --trust-store "ca:myCACerts" --trust-identity "x509.subject:C=US,ST=WA,O=wabbit-network.io"
+```
+
+Sample output for a successful initialization:
+
+```json
+{
+  "version": "1.0",
+  "trustPolicies": [
+    {
+        "name": "myBlobPolicy",
+        "signatureVerification": {
+          "level" : "strict"
+        },
+        "trustStores": ["ca:myCACerts"],
+        "trustedIdentities": [
+          "x509.subject:C=US,ST=WA,O=wabbit-network.io"
+        ]
+    }
+  ]
+}
+```
+
+Sample output for a failed initialization:
+
+```text
+Failed to initialize the blob trust policy configuration: <Reason>. <Recommendation>
+```
+
+For a `ca` type of trust store, the `--trust-store` value follows the format `ca:<store_name>`, while the `--trust-identity` value follows the format `x509.subject:<signing_cert_subject>`.
+
+The `--trust-store` and `--trust-identity` flags can be specified multiple times if multiple trust stores or trust identities need to be configured.
```

Comment:
Then, should the flags be named as `trust-stores` and `trust-identities`?

Reply:
it is a stringarray, for example `notation blob policy init --trust-store ca:mystore1 --trust-store ca:mystore2`, each one will has only one store, not multiple stores.